Bench-Kitura-Core/latest/Package.swift

```diff
@@ -19,10 +19,10 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.2.0"),
-        .package(url: "https://github.com/IBM-Swift/HeliumLogger.git", from: "1.0.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura-Session.git", from: "3.0.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura-Compression.git", from: "2.1.1"),
+        .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.4.0"),
```

Comment:
Do you think that changing the minimum minor/patch versions is necessary?  For example, there's nothing in these benchmarks (yet) that would require Kitura 2.4, they would compile fine on 2.2 or 2.3.  I'd prefer to update these if and when we introduce benchmarks that use the new APIs.